packages/notebook/src/browser/contributions/notebook-outline-contribution.ts

```diff
@@ -21,18 +21,20 @@ import { OutlineViewService } from '@theia/outline-view/lib/browser/outline-view
 import { NotebookModel } from '../view-model/notebook-model';
 import { OutlineSymbolInformationNode } from '@theia/outline-view/lib/browser/outline-view-widget';
 import { NotebookEditorWidget } from '../notebook-editor-widget';
-import { NotebookCellModel } from '../view-model/notebook-cell-model';
-import { DisposableCollection, URI } from '@theia/core';
+import { DisposableCollection, isObject, URI } from '@theia/core';
 import { CellKind, CellUri } from '../../common';
 import { NotebookService } from '../service/notebook-service';
 export interface NotebookCellOutlineNode extends OutlineSymbolInformationNode {
-    notebookCell: NotebookCellModel;
     uri: URI;
 }
 
 export namespace NotebookCellOutlineNode {
     export function is(element: object): element is NotebookCellOutlineNode {
-        return TreeNode.is(element) && OutlineSymbolInformationNode.is(element) && 'notebookCell' in element;
+        return TreeNode.is(element)
+            && OutlineSymbolInformationNode.is(element)
+            && isObject<NotebookCellOutlineNode>(element)
+            && typeof element.uri === 'object'
+            && (element as NotebookCellOutlineNode).uri.scheme === CellUri.cellUriScheme;
```

Comment:
Minor: The casting is already done by `isObject`
```suggestion
            && element.uri instanceof URI
            && element.uri.scheme === CellUri.cellUriScheme;
```